frontend/app/js/script.js

```diff
@@ -29,11 +29,18 @@ $(function() {
     $("a[data-action='enlarge']",codeDiv).show();
   });
 
-  $('form input[type=submit], form button[type=submit]').on('click', function() {
-    var $this = $(this);
+  $('form').on('submit', function() {
+    var $this = $(this).find('button');
```

Comment:
I get what it's doing, my point is that the old implementation was catching the event on both `button` and `input[type="submit"]` elements, but now it doesn't catch the latter. I would use `ack` to find any forms and make sure that there aren't any that are using `<input type="submit">` instead of `<button>`. Or just find both on line 33 here.